package.json

```diff
@@ -18,13 +19,20 @@
     "url": "https://github.com/mozilla/fxa-pairing-tls/issues"
   },
   "homepage": "https://github.com/mozilla/fxa-pairing-tls#readme",
+  "dependencies": {
+    "@babel/runtime": "^7.2.0"
```

Comment:
Do you want the "^" here? In all other repos we hard pin to particular versions. I'm not against the change, and think we should discuss whether the rationale for pinning to particular versions in other repos still holds.